### DISCUSSION TITLE
关于全屏界面的还原按钮

### DISCUSSION THREAD

**Discussion by tgjqr** [hooke007/mpv_PlayKit#501] [mpv 通用]
系统为Windows 11
在全屏模式下点击还原按钮（位于右上角三键中间位置）
得到
![3](https://github.com/user-attachments/assets/a7601f22-656e-4501-be2e-e8d95d765ff7)

此时再次点击还原按钮，得到效果类似全屏窗口化，并非`autofit`中指定的大小

若在出现方角矩形时（图片状态）点击右下角的“切换全屏”，再在全屏模式下点击还原，方角矩形变回圆角矩形

下面是mpv中关于窗口的设置
```
 force-window = immediate
 autofit-smaller = 40%x40%
 autofit-larger = 80%x80%
 keepaspect-window = no
 hidpi-window-scale = no
```

**Comment by hooke007** [comment]
切换全屏返回的始终是上个窗口的状态
autofit一般也只在初始化，值变更，vid刷新的时候触发窗口进行对应变化。

所以问题到底是什么

**Reply by hooke007** [reply]
另外直角圆角不稳定变化这个大概率是win11自己杂交ui的bug

**Reply by tgjqr** [reply]
问题是图片状态已经是还原状态，因此这里的按钮不应该是还原，而应该是最大化

**Reply by hooke007** [reply]
你说的还原按钮是哪个？右上角那三个键在全屏状态根本不会存在，要怎么在（在全屏模式下点击还原按钮（位于右上角三键中间位置））？

**Reply by hooke007** [reply]
我大概理解你的问题了。
你看清楚：全屏绘制的按钮是uosc的界面，不是windows的按钮
uosc中间的按钮在windows上是全屏切换功能
